webapp/platform/client/src/websocket.test.ts

```diff
@@ -418,4 +459,160 @@ describe('websocketclient', () => {
 
         jest.useRealTimers();
     });
+
+    test('should add network event listener on initialize', () => {
```

Comment:
I wonder if we should add tests for the "window is undefined" case.

Reply:
I dropped the "window is undefined" logic